### PR TITLE
task/0053-CW All Star Code Challenge #1 

### PR DESCRIPTION
**Pull request by Dimkaze**

### CODE REVIEW

**Comment by mTeeeur** [comment]
@Dimkaze why do you remove some files?

File: tasks/StarCode/sumPPGagain.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function sumPPG(playerOne, playerTwo) {
+  return playerOne.ppg + playerTwo.ppg;
+}
```
Comment (mTeeeur) [inline tasks/StarCode/sumPPGagain.js:3]: we can use ```() => {}``` lambda expression here

File: tasks/task-0053-C-All-Star-CodeChallenge-#1 [hunk 2]
```diff
@@ -0,0 +1 @@
+const sumPPG = (playerOne, playerTwo) => playerOne.ppg + playerTwo.ppg;
```
Comment (mTeeeur) [inline tasks/task-0053-C-All-Star-CodeChallenge-#1:1]: why do you add this file?

**Review by mTeeeur — CHANGES_REQUESTED** [review]

File: tasks/StarCode/sumPPGagain.js (hunk 1 above)
Comment (Dimkaze) [reply tasks/StarCode/sumPPGagain.js:3]: When i did this function on arrow back call, test fuck me and say than const sumPPG is not a function

File: tasks/task-0053-C-All-Star-CodeChallenge-#1 (hunk 2 above)
Comment (askomar) [reply tasks/task-0053-C-All-Star-CodeChallenge-#1:1]: Me too. Why are you add unnesesary file?

**Review by askomar — CHANGES_REQUESTED** [review]
Do rebase to the last commit of dev. And you will not need to delete my files. And why do you try to delete sum folder?

And your commit naming is not correct. It must be:  task/cw-0001 init project
Так должен именоваться коммит
 Первое что за задача
 Потом номер ветки
 И описание

**Pull request closed without merging**